index.html

```diff
@@ -259,6 +269,36 @@
         <div class="prepend-input">💣</div>
         <input id="bombs" class="input" type="number" value="10">
 
+        <select id="theme" onchange="_change(this);" class ="box">
+            <option selected value="Normal">Normal🐣</option>
+            <option value="Flag">Flag🇺🇸</option>
+            <option value="Fruits">Fruits🍒</option>
+            <option value="Celebration">Celebration🎁</option>
+        </select>
+
+        <script>
+      $(document).ready(function() {
```

Comment:
There is no jQuery on the site so this actually doesn't work. (I'd prefer not adding jQuery here)